buildSrc/src/BuildThemes.ts

```diff
@@ -627,6 +627,11 @@ function getName(dokiDefinition: MasterDokiThemeDefinition) {
   return dokiDefinition.name.replace(':', '');
 }
 
-function isKanna(dokiDefinition: { colors: StringDictionary<string>; id: string; name: string; displayName: string; dark: boolean; author: string; group: string; overrides?: any | undefined; product?: "community" | "ultimate" | undefined; stickers: any; editorScheme?: any | undefined; }): boolean {
-  return dokiDefinition.id === "b93ab4ea-ff96-4459-8fa2-0caae5bc7116"
+const excludedSecondaryContentThemes = new Set([
+ 'b93ab4ea-ff96-4459-8fa2-0caae5bc7116', // Kanna's are the same
+  'ea9a13f6-fa7f-46a4-ba6e-6cefe1f55160' // I lack culture...
+]);
```

Comment:
There is an issue with Kanna's background not showing in the *new tab* page.

https://user-images.githubusercontent.com/23222943/148717461-9e384d5f-a8b7-4e3b-bae8-0a085596c4b7.mp4

When issue was found, I've already looked through the rest of the themes to make sure none of them have the same problem.

Reply:
Good catch! I am looking forward to refactoring the BuildThemes.ts when pulling out firefox.

Reply:
Yep, this resolves it!